jobs-shell/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloudrun_jobs_shell_dockerfile]
+
+# Use the official Ubuntu image from Docker Hub as
+# a base image
+FROM ubuntu:bionic
+
+# Copy over the script to the /workspace directory
+COPY script.sh /workspace/
+
+# Execute next commands in the directory /workspace
+WORKDIR /workspace
+
```

Comment:
Do you think we need a backup `RUN chmod +x ./script.sh`?

Reply:
Why do you think we need that?

Reply:
It can cause friction if the script isn't executable and sometimes those permissions get mixed up